components/hab/src/license.rs

```diff
@@ -152,19 +174,21 @@ fn writeable_license_path() -> PathBuf {
     license_path(&root_dir)
 }
 
-fn env_var_present() -> Result<bool> {
+fn env_var_present() -> Result<LicenseAcceptance> {
```

Comment:
Might be better to name this something like `acceptance_from_env_var` now.

Reply:
Agreed, thanks! Addressed in [2c27f4](https://github.com/habitat-sh/habitat/pull/7834/commits/2c27f45f18ef19704aaf768198df3517c2e0e273)